apps/fishing-map/features/workspace/shared/DatasetSchemaField.tsx

```diff
@@ -14,7 +14,23 @@ type LayerPanelProps = {
 }
 
 function DatasetSchemaField({ dataview, field, label }: LayerPanelProps): React.ReactElement {
-  const valuesSelected = getSchemaFieldsSelectedInDataview(dataview, field)
+  let valuesSelected = getSchemaFieldsSelectedInDataview(dataview, field).sort(
+    (a, b) => a.label - b.label
+  )
+
+  const valuesAreRangeOfNumbers =
+    valuesSelected.length > 1 && valuesSelected.every((value) => Number(value.label))
```

Comment:
Shouldn't you use isNan or typeof here?

Reply:
I think we are good like this because the method `Number` returns:
- 0 for ''
- NaN for 'asdf'
- 1 for '1'